conjure-java-jersey-server/src/test/java/com/palantir/conjure/java/server/jersey/Java8OptionalTest.java

```diff
@@ -102,6 +102,12 @@ public void testQueryParam_optionalIntEmpty() {
         assertThat(response.readEntity(String.class), is("0"));
     }
 
+    @Test
+    public void testQueryParam_optionalIntInvalid() {
+        Response response = target.path("optional/int").queryParam("value", "foo").request().get();
```

Comment:
Do we want an explicit test for empty string? Would expect that one to be the most likely to be accidentally "fixed"